src/main/resources/application.yaml

```diff
@@ -248,6 +248,11 @@ internalDetainedRequestRespondentEvidence.fileExtension: PDF
 internalDetainedRequestRespondentEvidence.fileName: "detained-appellant-request-respondent-evidence-notice"
 internalDetainedRequestRespondentEvidence.templateName: ${IA_INTERNAL_DETAINED_REQUEST_RESPONDENT_EVIDENCE_TEMPLATE:TB-IAC-GNO-ENG-00010.docx}
 
+internalDetainedEndAppealAutomatically.contentType:  application/pdf
+internalDetainedEndAppealAutomatically.fileExtension: PDF
+internalDetainedEndAppealAutomatically.fileName: "detained-appellant-request-respondent-evidence-notice"
```

Comment:
the fileName is not reflecting the document properly.

Reply:
Fixed this one as well